src/sagemaker_tensorflow_container/training.py

```diff
@@ -138,7 +142,10 @@ def train(env):
 
         if not _is_host_master(env.hosts, env.current_host):
             _wait_until_master_is_down(env.hosts[0])
-
+    elif mpi_enabled:
+        logger.info("Running training job with MPI.")
+        framework.entry_point.run(env.module_dir, env.user_entry_point,
+                                  env.to_cmd_args(), env.to_env_vars(), mpi_enabled=mpi_enabled)
```

Comment:
Do we need the statement below?

Reply:
Can you please elaborate? What exactly we dont need it ?

Reply:
The logic in the conditional was repeated and the code could have being refactored.